core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -66,6 +66,10 @@ oppia.controller('FeedbackTab', [
     $q.all([userInfoPromise, threadPromise]).then(function() {
       $rootScope.loadingMessage = '';
     });
+    $scope.fetchAgain = function() {
+      ThreadDataService.fetchThreads();
+      $scope.threadData = ThreadDataService.data;
+    };
```

Comment:
Now I think you're fetching the feedback threads from the database every time the creator clicks 'Back' button in the thread, right? As you stated previously, they were being fetched only when a new thread comes in.

So, instead of fetching every time when a back button is clicked, can we fetch them only when some thread is updated (probably at the time of clicking the 'Save' button)? What do you think?

Reply:
Yeah this will be good.

Reply:
Done

Reply:
Actually I tried binding the function with the 'Send' button, but it was causing issues as at the same time message updating and fetching threads function were called, so I came up with another alternative.

Reply:
Yeah, it sounds good!